src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -1218,8 +1218,36 @@ public class FlowStateAllowedEventService {
          entry(
             PART_ADMIT_NOT_SETTLED_NO_MEDIATION.fullName(),
             List.of(
+                ADD_DEFENDANT_LITIGATION_FRIEND,
+                ENTER_BREATHING_SPACE_SPEC,
+                ENTER_BREATHING_SPACE_LIP,
+                LIFT_BREATHING_SPACE_SPEC,
+                WITHDRAW_CLAIM,
+                DISCONTINUE_CLAIM,
+                CASE_PROCEEDS_IN_CASEMAN,
+                AMEND_PARTY_DETAILS,
+                INITIATE_GENERAL_APPLICATION,
                 CREATE_SDO,
+                NotSuitable_SDO,
```

Comment:
How come this is not in all capitals like other events?

Reply:
When SDO created this event, they for some reason decided not to capitalise it.
I did not change this as part of my ticket.

Reply:
{
    "CaseTypeID": "CIVIL",
    "ID": "NotSuitable_SDO",
    "Name": "Not suitable for SDO",
    "Description": "Not suitable for SDO",
    "DisplayOrder": 20,
    "PreConditionState(s)": "JUDICIAL_REFERRAL",
    "PostConditionState": "PROCEEDS_IN_HERITAGE_SYSTEM",
    "SecurityClassification": "Public",
    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/submitted",
    "ShowSummary": "Y",
    "ShowEventNotes": "N",
    "EndButtonLabel": "Submit",
    "RetriesTimeoutAboutToStartEvent": 0,
    "RetriesTimeoutURLAboutToSubmitEvent": 0,
    "Publish": "Y"
  }